src/LazyOperations/ConvexHull.jl

```diff
@@ -3,9 +3,7 @@ import Base.isempty
 export ConvexHull, CH,
        convex_hull,
        convex_hull!,
-       ConvexHullArray, CHArray,
        ConvexHull!,
-       array,
        swap
```

Comment:
This is a method of the `CHArray` (but it does not matter where we write the `export`s).

Reply:
> swap is a method of the CHArray

Are you sure?

Reply:
Nah, that was incorrect.